src/transformers/activations_tf.py

```diff
@@ -15,7 +15,20 @@
 import math
 
 import tensorflow as tf
-from packaging import version
+from packaging.version import parse
+
+
+try:
+    import tf_keras as keras
+except (ModuleNotFoundError, ImportError):
+    import keras
+
+    if parse(keras.__version__).major > 2:
+        raise ValueError(
+            "Your currently installed version of Keras is Keras 3, but this is not yet supported in "
+            "Transformers. Please install the backwards-compatible tf-keras package with "
+            "`pip install tf-keras`."
+        )
```

Comment:
I'm not sure I understand the rational for when the version compatible import is just handled e.g. in `src/transformers/keras_callbacks.py` and when the version is checked e.g. here

Reply:
I've rewritten things to always import from `modeling_tf_utils`. The only exceptions, where I copy-pasted it instead, are here in `activations_tf`, because that would create a circular import, and in the example files, which are designed for users to read and modify so I want to make it clear exactly which `keras` they're getting.

Reply:
Perfect :)